capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -21,17 +20,12 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
+    // API Key for Civic API access
+    String Key_Test = "";
```

Comment:
nit while you're editing this file, can you update the variable name to keyTest?

Reply:
Sure thing